ubuntu22.04/precompiled/nvidia-driver

```diff
@@ -276,6 +320,10 @@ _unmount_rootfs() {
 }
 
 init() {
+    _fetch_nvidia_installer
```

Comment:
Why are we doing this at runtime? Shouldn't we be fetching the `nvidia-installer` at build time and including the binary in the image? Ideally the precompiled image should be entirely self-contained.

Reply:
Agreed. I'll move this to the build time